### ISSUE TITLE
Add Open Graph tags

### ISSUE THREAD

**Issue by timmalstead** [hackforla/food-oasis#456]
### Overview
We should have open graph tags and/or twitter tags for our site.

### Action Items
- [ ] figure out which images and copy to use
- [ ] make and implement tags
